packages/create-svelte/templates/default/src/routes/todos/_api.ts

```diff
@@ -18,7 +18,7 @@ const base = 'https://api.svelte.dev';
  */
 export function api(method: string, resource: string, data?: Record<string, unknown>) {
 	return fetch(`${base}/${resource}`, {
-		method,
+		method: method.toUpperCase(),
```

Comment:
... This makes me uncomfortable. Does undici require uppercase method names while node-fetch didn't? Needing this feels like a workaround, and I'd prefer trying to get to the bottom of what changed rather than fixing it here in the template project. We shouldn't expect everyone using SvelteKit to always be calling their APIs with upper-case `method`s.

Reply:
Hmm, I think it was just patch that was 400-ing, so I didn't consider the switch from node-fetch to undici as a possible cause, but not completely off the table I guess. I'll do a quick sanity check.